pincer/oggparse.py

```diff
@@ -0,0 +1,96 @@
+"""
+This is a modified version of discord.py's OggParse module
```

Comment:
Incorrect liscence header. Look at any other file in the lib and copy the style for all of your files.